src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,34 @@
+import { useContext, useState, useEffect} from 'react';
+import { useLocation } from 'react-router-dom';
+
+import { TodoItem } from '../TodoItem';
+
+import { Context } from '../ContextProvider/ContextProvider';
+import { Todo } from '../../types/Todo';
+
+export const TodoList = () => {
+  const [filteredTodos, setFilteredTodos] = useState<Todo[]>([]);
```

Comment:
it's redundant to add the state for filtered todos, you can just use the useMemo or some function to filter them and save in simple var